src/agent/agent.c

```diff
@@ -532,6 +539,7 @@ static int agent_run_unit_lifecycle_method(sd_bus_message *m, Agent *agent, cons
  ************************************************************************/
 
 static int agent_method_start_unit(sd_bus_message *m, void *userdata, UNUSED sd_bus_error *ret_error) {
+        hirte_log_debug("Starting unit");
```

Comment:
If you move this log to 
https://github.com/containers/hirte/blob/main/src/agent/agent.c#L500-L502
we can have this debug log for starting, stopping, etc. After reading the message, a few lines below, we could even log the unit name.

Reply:
going to just drop that line